frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { cleanup, render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { afterEach, expect, it } from 'vitest'
+import { CommandPalette } from './CommandPalette'
+
+afterEach(() => {
+  cleanup()
```

Comment:
It executes `cleanup` function after each test, otherwise the dialog component will keep displayed.

https://testing-library.com/docs/react-testing-library/api/#cleanup